tests/Driver/VersionAwarePlatformDriverTest.php

```diff
@@ -73,7 +74,7 @@ public static function mySQLVersionProvider(): array
                 'https://github.com/doctrine/dbal/pull/5779',
                 false,
             ],
-            ['mariadb-10.9.3', MariaDB1052Platform::class, 'https://github.com/doctrine/dbal/pull/5779', true],
+            ['mariadb-10.9.3', MariaDb1060Platform::class, 'https://github.com/doctrine/dbal/pull/5779', true],
```

Comment:
We might want to add a test that makes sure, we pick `MariaDb1060Platform` for a MariaDB 10.6 server.

Reply:
added the test case